docs/pages/User/GettingStarted/demo_k8s_on_desktop.md

```diff
@@ -0,0 +1,216 @@
+
+![](/images/k8s-on-desktop-demo/introduction_schema.svg)
+
+Obviously, for the purpose of the demo it would be better if the remote machine features an NVIDIA graphic card, while the local machine can be a traditional laptop.
+To be more specific, we will execute a Blender `pod` in a *foreign cluster* (that is represented in the *local cluster* as a *virtual node* named `liqo-<...>`) and a viewer `pod` in the *local cluster*.
+
+**Note**: from now on, when we'll talk about "*local cluster*" we'll refer to the one that will run the `cloudify` script ([see afterwards](#about-kubernetesondesktop)), and when we'll talk about "*foreign cluster*" we'll refer to the other one.
+
+
+## The KubernetesOnDesktop project
+[KubernetesOnDesktop](https://github.com/netgroup-polito/KubernetesOnDesktop) (KoD) aims at developing a cloud infrastructure to run desktop applications in a remote Kubernetes cluster.
+In a nutshell, KoD splits traditional desktop applications in a backend (running the actual application) and a frontend, running the graphical interface and interacting with the (desktop) user.
+This enables dektop applications to be executed also on a remote machine, while keeping their GUI locally.
+
+Technically, KoD leverages a client/server VNC+PulseAudio+SSH infrastructure that enables to start the application `pod` in a k8s remote node and redirects its GUI (through VNC) and the audio (through PulseAudio+SSH) in a second `pod` scheduled in the node where the `cloudify` application is running.
+The communication between the two components leverages several kubernetes primitives, such as `deployments`, `jobs`, `services` (particularly, `ClusterIP` is used) and `secrets`.
+For further information see [KubernetesOnDesktop](https://github.com/netgroup-polito/KubernetesOnDesktop) GitHub page.
+
+So far, KoD suppors firefox, libreoffice and blender, the latter with the capability to exploit any available NVIDIA GPUs (through the NVIDIA CUDA driver) if the remote node has this hardware.
+In any case, thanks to the massive use of templates, many more applications can be easily supported.
+
+When executing the `cloudify` command, the application will create:
+  * a `secret` containing an ssh key that allows the two application components to communicate securely;
+  * a `deployment` containing the application (e.g. blender) and the VNC server, whose `pod` will be scheduled on the remote cluster;
+  * a `ClusterIP` `service` that allows the remote `pod` to be reachable from other `pod`s in the cluster by using [K8s DNS for services](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/);
+  * a `pod` executing the VNC viewer, started in the local machine (i.e., on the same node where you run `cloudify`).
+  
+## Installation of the required software
+To install all the required software we need to follow this steps:
+
+1. [Install k3s](#install-k3s) in both clusters;
+2. [Install liqo](#install-liqo) in both clusters; 
```

Comment:
@riccardoroccaro It seems to me that we always wrote liqo with the capital letter (i.e., Liqo). Please inspect this file and change all occurrencies of "liqo" accordingly (only when liqo is a name, not part of something else, e.g., liqo-<...> such as the name of the pod). Thanks!

Reply:
Fixed